spyder_remote_services/services/fsspec/handlers.py

```diff
@@ -0,0 +1,188 @@
+            )
+        else:
+            self.log.warning(
+                "Uncaught exception %s\n%r",
+                self._request_summary(),
+                self.request,
+                exc_info=(typ, value, tb),  # type: ignore
+            )
+
+class LsHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def get(self, path):
+        detail_arg = self.get_argument("detail", default="true").lower()
+        detail = detail_arg == "true"
+        result = self.fs_ls(path, detail=detail)
+        self.write_json(result)
+
+
+class InfoHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def get(self, path):
+        result = self.fs_info(path)
+        self.write_json(result)
+
+
+
+class ExistsHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def get(self, path):
+        result = self.fs_exists(path)
+        self.write_json({"exists": result})
+
+
+class IsFileHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def get(self, path):
+        result = self.fs_isfile(path)
+        self.write_json({"isfile": result})
+
+
+class IsDirHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def get(self, path):
+        result = self.fs_isdir(path)
+        self.write_json({"isdir": result})
+
+
+class MkdirHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def post(self, path):
+        create_parents = (self.get_argument("create_parents", "true").lower() == "true")
+        exist_ok = (self.get_argument("exist_ok", "false").lower() == "true")
+        result = self.fs_mkdir(path, create_parents=create_parents, exist_ok=exist_ok)
+        self.write_json(result)
+
+
+class RmdirHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def delete(self, path):
+        result = self.fs_rmdir(path)
+        self.write_json(result)
+
+
+class RemoveFileHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def delete(self, path):
+        missing_ok = (self.get_argument("missing_ok", "false").lower() == "true")
+        result = self.fs_rm_file(path, missing_ok=missing_ok)
+        self.write_json(result)
+
+
+class TouchHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def post(self, path):
+        truncate = (self.get_argument("truncate", "true").lower() == "true")
+        result = self.fs_touch(path, truncate=truncate)
+        self.write_json(result)
+
+
+class CopyHandler(BaseFSSpecHandler):
+    @web.authenticated
+    @authorized
+    def post(self, path):
+        dest = re.match(_path_regex, self.get_argument("dest")).group("path")
+        metadata = (self.get_argument("metadata", "false").lower() == "true")
+        result = self.fs_copy(path, dest, metadata=metadata)
+        self.write_json(result)
+
+
+_path_regex = r"file://(?P<path>.+)"
+
+handlers = [
+    (rf"/fsspec/open/{_path_regex}", ReadWriteWebsocketHandler),  # WebSocket
```

Comment:
`/fsspec/open/` -> `/fs/open/` and the same for the other routes below this one.